### DISCUSSION TITLE
zh/patterns/

### DISCUSSION THREAD

**Discussion by giscus** [iluwatar/java-design-patterns-vuepress-web#74] [Announcements]
# zh/patterns/

Abstract Document Abstract Factory Active Object Acyclic Visitor Adapter Aggregator Microservices Ambassador API Gateway Arrange/Act/Assert Async Method Invocation Balking Bridge Builder Business Delegate Bytecode Caching Callback Chain of responsibility Circuit Breaker Static Content Hosting Collection Pipeline Command Composite Composite Entity Converter Data Access Object Data Bus Data Mapper Data Transfer Object Decorator Delegation Dependency Injection Dirty Flag Double Checked Locking Facade Factory Factory Kit Factory Method Interpreter Iterator Observer Private Class Data Producer Consumer Proxy Sharding Singleton State Strategy Template method Version Number Visitor

https://java-design-patterns.com/zh/patterns/

**Comment by springboot4** [comment]
very good

**Comment by jason1024po** [comment]
中文版不显示目录呀

**Reply by jason1024po** [reply]
Mac OS Chrome Version 133.0.6943.126 (Official Build) (arm64)

**Comment by AnNingUI** [comment]
不错，非常有学习价值
